### ISSUE TITLE
After upgrading to 15.* the menubar list is not correctly showing the current and non-empty workspaces

### ISSUE THREAD

**Issue by nmder** [nikitabobko/AeroSpace#586]
**After upgrading to 15.* the menubar list is not correctly showing the current and non-empty workspaces**

<img width="293" alt="image" src="https://github.com/user-attachments/assets/aa311a51-5b21-4d76-8337-9a7c72e57c95">
 
As you can see in the attachment, the current workspace is 7, as opposed to indicated by the list 0. In fact, the active workspace never changes from 0 in the list.

The non-empty workspaces D and B are not marked as well. Notably, windows were moved to D and B from 0.

## Checklist

- [x] I've searched for duplicates. My search queries were (please don't forget to search in closed issues too):
- [x ] I've read the guide https://nikitabobko.github.io/AeroSpace/guide
- [x] I've searched in the documentation (https://nikitabobko.github.io/AeroSpace/guide & https://nikitabobko.github.io/AeroSpace/commands). My search queries were:

```
aerospace CLI client version: 0.15.2-Beta b6cf82771f245ab7349a93baf8709e171537ff58
AeroSpace.app server version: 0.15.2-Beta b6cf82771f245ab7349a93baf8709e171537ff58
```

**Comment by nikitabobko** [comment]
2 days ago I opened the menu bar and I had the same problem as you describe

Now, I can't reproduce the problem 🤷

**Comment by nmder** [comment]
Is it the same version of aerospace?

**Comment by nikitabobko** [comment]
Yes, 0.15.2, I assume that the problem is still reproducible for you

**Comment by nmder** [comment]
> Yes, 0.15.2, I assume that the problem is still reproducible for you

It is..

**Comment by garrettwp** [comment]
I experience the same issue, also alt-tabing between workspaces will skip a workspace. For example, If I am on workspace 1 and have windows on workspace 2, pressing (option) alt-tab will go to workspace 3 and skip 2.

**Comment by romanskorobagatko** [comment]
@garrettwp I had the same issue a few times, v0.15.2-Beta b6cf8277.

**Comment by nikitabobko** [comment]
Should be fixed in 0.16.0-Beta
